packages/shared/src/components/search/SearchContainer.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactNode } from 'react';
+import { SearchBar } from './SearchBar';
+import { SearchFeedback } from './SearchFeedback';
+
+interface SearchContainerProps {
+  children: ReactNode;
+  onSubmit(event: React.MouseEvent, value: string): void;
+}
+
+export function SearchContainer({
+  children,
+  onSubmit,
+}: SearchContainerProps): React.ReactElement {
+  return (
+    <main className="m-auto w-full">
+      <div className="grid grid-cols-1 laptop:grid-cols-3 gap-y-6 pt-8 m-auto max-w-screen-laptop">
```

Comment:
Is the grid necessary here?
If not prefer flex in case things dynamic render.

Reply:
Oh, this was the existing classes when I moved this container. Let me check if it is necessary.

Reply:
Seems okay with flex 🥳